tests/conftest.py

```diff
@@ -58,6 +63,91 @@
 from tests.utils import create_image
 
 
+class CaptureQueriesContext(BaseCaptureQueriesContext):
+    IGNORED_QUERIES = settings.PATTERNS_IGNORED_IN_QUERY_CAPTURES
+
+    @property
+    def captured_queries(self):
+        # flake8: noqa
+        base_queries = self.connection.queries[
+            self.initial_queries : self.final_queries
+        ]
+        new_queries = []
+
+        def is_query_ignored(sql):
+            for pattern in self.IGNORED_QUERIES:
+                # Ignore the query if matches
+                if pattern.match(sql):
+                    return True
+            return False
+
+        for query in base_queries:
+            if not is_query_ignored(query["sql"]):
+                new_queries.append(query)
+
+        return new_queries
+
+
+def _assert_num_queries(context, *, config, num, exact=True, info=None):
+    """
+    Extracted from pytest_django.fixtures._assert_num_queries
+    """
```

Comment:
I don't think this docstring is correct by pydocstyle.

Reply:
Tests are exclude from pydocstyle

Reply:
Well, it doesn't mean that we don't have to follow it.